components/shared/BalanceInput.vue

```diff
@@ -145,3 +146,11 @@ defineExpose({
   checkValidity,
 })
 </script>
+
+<style scoped lang="scss">
+:deep .h-full {
```

Comment:
we already have a `h-full` in global.scss, do you really need to add it there too?

Reply:
i use the one from `global.scss`
and I use it here to adjust the style of a child component
overriding oruga UI default height

![image](https://github.com/kodadot/nft-gallery/assets/22791238/72ab598a-b12a-4a84-bb03-c30e4153979c)

Reply:
I suggest adding a new class name instead of `h-full` to do this overwrite.

Reply:
`h-full` was not picked at random
it is used both for `height: 100%` and as utility for overwrite

![image](https://github.com/kodadot/nft-gallery/assets/22791238/9c952065-0dbf-42df-b953-b628e0c051c1)

Reply:
or we could unify size and make select box same size as default input (56px => 48px)

Reply:
sure. lets try that

Reply:
done
indeed clean and simple